### PR TITLE
Reformat the checklist to match the new style

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

**Comment by rebekaburnett** [comment]
Hello! We are refactoring all the checklists to be in categories and look less like a form that people need to fill out. Yours was different from the others, so a bit more challenging. Will you have a look and see what you think?

File: vipps-login-api-checklist.md [hunk 1]
```diff
@@ -11,46 +11,50 @@ pagination_prev: null
...
+| Do not share `client_secret`   | Ensure that the `client_secret` is kept secret and is never be shared to browsers or native apps. |
+
+
+## Quality assurance
+
+| Action | Comment |
+|--------|---------|
+|  Ensure functionality   | Ensure that your solution is verified to work if the user start in a "non-default" browser on mobile, e.g. start Vipps Login from Chrome browser on iOS. |
+| Update user registry    | Implement proper linking of the Vipps user to your own user registry. This login must be based on either phone number or e-mail address. See [recommendations on linking to user account](api-guide/important-information.md#Recommendations-on-linking-to-user-account). |
+|     Handle errors | Make sure to log and handle all errors. Cancelled logins and error situations on the redirect back to `redirect_uri` (redirects with a `error` query parameter) are handled. All integrations should to display errors in a way that the users (customers and merchant employees/administrators) can see and understand them.|
```
Comment (jonkjenn) [inline vipps-login-api-checklist.md:52]:
```suggestion
|     Handle errors | Make sure to log and handle all errors. Cancelled logins and error situations on the redirect back to `redirect_uri` (redirects with a `error` query parameter) are handled. All integrations should display errors in a way that the users (customers and merchant employees/administrators) can see and understand them.|
```

Comment (jonkjenn) [inline vipps-login-api-checklist.md:52]: Something strange with these sentences
```
|     Handle errors | Make sure to log and handle all errors. Cancelled logins and error situations on the redirect back to `redirect_uri` (redirects with a `error` query parameter) are handled.
```

File: vipps-login-api-checklist.md [hunk 2]
```diff
@@ -11,46 +11,50 @@ pagination_prev: null
...
+
+## Integration considerations
+
+| Action | Comment   |
+|--------|-----------|
+| Use correct flow   | Native app integrations use the [app-to-app flow](api-guide/overview.md#app-to-app-flow). [Vipps Login from phone number](api-guide/overview.md#vipps-login-from-phone-number) must _not_ be used for web-based login.|
+| Do not use embedded iFrames | Integration with the Vipps Login API is redirect-based (i.e., no embedded iFrame). |
+|  Set company name and logo  | The sales unit name appears on the Vipps Login landing page. Both the name and logo appear in the Vipps app under _Personal information_ > _Companies with access_. See [how to change name and logo](vipps-login-api-faq.md#how-can-i-change-my-name-and-logo).|
+|   Whitelist redirect URIs   | Ensure that all `redirect_uri`s for your integration have been added using to the merchant portal white list. See the [FAQ](vipps-login-api-faq.md#how-can-i-activate-and-set-up-vipps-login) for how to do this. Ensure that all URLs use `HTTPS` or native URL schemes (i.e., `vipps://`), not `HTTP`. ALL URLs must match the exact URI sent on `/auth` request. No query parameters or additional trailing `/` can be included. |
```
Comment (jonkjenn) [inline vipps-login-api-checklist.md:32]:
```suggestion
|   Whitelist redirect URIs   | Ensure that all `redirect_uri`s for your integration have been added to the merchant portal white list. See the [FAQ](vipps-login-api-faq.md#how-can-i-activate-and-set-up-vipps-login) for how to do this. Ensure that all URLs use `HTTPS` or native URL schemes (i.e., `vipps://`), not `HTTP`. ALL URLs must match the exact URI sent on `/auth` request. No query parameters or additional trailing `/` can be included. |
```

File: vipps-login-api-checklist.md (hunk 1 above)
Comment (rebekaburnett) [reply vipps-login-api-checklist.md]: Yes, you're right. I will address it. :-)

Comment (rebekaburnett) [reply vipps-login-api-checklist.md]: Make sure to log and handle all errors. For example, handle cancelled logins and error situations while redirecting the user back to redirect_uri (i.e., redirect with an error query parameter). In addition, display errors in a way that the users (customers and merchant employees/administrators) can see and understand them.

**Review by jonkjenn — APPROVED** [review]
:+1:

**Pull request merged**